.github/workflows/process_label_removal.yml

```
@@ -0,0 +1,24 @@
+name: Process label removal
+on: 
+  pull_request:
+    types:
+      - unlabeled
+
```

```suggestion
permissions:
  contents: read
```
This is for https://datadoghq.atlassian.net/browse/VULN-8298. I don't see anything here that needs write permissions, but I could be wrong